### ISSUE TITLE
`suite.load('cheetah', 'run')` breaks with mujoco 3.0

### ISSUE THREAD

**Issue by vmoens** [google-deepmind/dm_control#430]
Hello!

When executing 
```
$ pip install dm_control
$ python -c """
from dm_control import suite

suite.load('cheetah', 'run')
"""
```
The following error is raised

```
AttributeError: 'MjModel' object has no attribute 'bvh_geomid'
```
Which is due to the recent release of mujoco 3.0 I believe

**Comment by saran-t** [comment]
I just uploaded version 1.0.15. Could you please try again?

**Comment by vmoens** [comment]
Fixed thanks a mil! And congrats on v3.0, awesome to see that Mujoco now works on device
